src/ucar/unidata/apps/trex/catalog.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<catalog xmlns="https://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0" xmlns:xlink="http://www.w3.org/1999/xlink" name="TREX Catalog">
```

Comment:
And again, the Unidata and XLink URIs ...
I won't comment on any remaining ones. Probably all the "xmlns" attributes should be reviewed.